keep/api/routes/preset.py

```diff
@@ -22,15 +23,23 @@ def get_presets(
 ) -> list[PresetDto]:
     tenant_id = authenticated_entity.tenant_id
     logger.info("Getting all presets")
-    statement = select(Preset).where(Preset.tenant_id == tenant_id)
+
+    #only global presets
+    statement = select(Preset).where(Preset.tenant_id == tenant_id, Preset.created_by == None)
     presets = session.exec(statement).all()
+
+    #private to user
+    statement = select(Preset).where(Preset.tenant_id == tenant_id, Preset.created_by == authenticated_entity.email)
+    presets.extend(session.exec(statement).all())
```

Comment:
why 2 queries?
```suggestion
    statement = select(Preset).where(Preset.tenant_id == tenant_id)
    presets = session.exec(statement).all()
```

Reply:
@talboren won't it also `select(Preset).where(Preset.tenant_id == tenant_id)` will rereive all presets created inside this tenant including the private ones of other users in the same tenant

won't it be wrong?

Reply:
But this is exactly what you are doing:
1. You get all the presets of specific tenant, where `created_by == None` and put them into `presets` variable
2. You get all the presets of specific tenant, where `created_by == authenticated_entity.email` and extend the `presets` variable with the results
This yields a `presets` variable that contains exactly what you just said.

Either I'm not understanding something or this is just the same.

Reply:
Refer this table:

preset_id | created_by    |  tenant_id
1               |   a@xyz.com  | 1
2               |   None             | 1
3               |   b@xyz.com  | 1

Query1: will yeild ids [2]
Query2: will yeild ids [1]

that makes it `presets = [1,2]`

If we only do `select(Preset).where(Preset.tenant_id == tenant_id)`, we will get => `[1,2,3]`.

Did I understood the problem statement wrong? I was assuming that preset created by a user will not be visible other users. Am I wrong here?

@talboren let me know

Reply:
@talboren or else you want this logic to be handled on frontend? 
so backend will return all presets?

Reply:
Gotcha! Seems like your logic is right but can still be achieved with 1 query instead of 2.
```
.where(Preset.tenant_id == tenant_id).
where(Preseet.created_by == None or Preset.created_by == email)
```
* not sure about the syntax ^ :P